### PR TITLE
OCPBUGS-27959: fix panic in health check timeouts

### PR DESCRIPTION
**Pull request by tjungblu**

### CODE REVIEW

File: pkg/etcdcli/health.go [hunk 1]
```diff
@@ -52,10 +52,13 @@ func getMemberHealth(ctx context.Context, etcdMembers []*etcdserverpb.Member) me
...
 		go func() {
-			ctx, cancel := context.WithTimeout(ctx, defaultTimeout)
+			ctxTimeout, cancel := context.WithTimeout(ctx, defaultTimeout)
 			defer cancel()
 
-			resChan <- checkSingleMemberHealth(ctx, member)
+			resChan <- checkSingleMemberHealth(ctxTimeout, member)
+			// closing here to avoid late replies to panic on resChan,
+			// the result will be considered a timeout anyway
+			close(resChan)
```
Comment (Elbehery) [inline pkg/etcdcli/health.go:61]: can we comment the issue here for the future ?

Comment (tjungblu) [reply pkg/etcdcli/health.go:61]: it's in the git commit :+1:

**Comment by tjungblu** [comment]
fixing a panic I think we can justify setting:

/label acknowledge-critical-fixes-only

/cherry-pick release-4.15

**Pull request merged**